### DISCUSSION TITLE
CRUD支持部分字段不提供

### DISCUSSION THREAD

**Discussion by ghost** [rbatis/rbatis#269] [General]
我想用PG的自增id主键，所以我不需要id字段，save的时候，怎么排除这不要的id 字段

**Comment by ghost** [comment]
我理解的，当定义的模型如果是Null，那你就别提供这个字段，这样插入的时候，就排除掉这个字段。

或者说你给我一个macro注解

```rust
#[insert_ommit_none]
pub id: Option(i64),
```

**Comment by zhuxiujia** [comment]
> 

可以使用传入Skip选项，save(arg,Skip::Column("id"))

**Comment by ghost** [comment]
@zhuxiujia , postgres还是有null 字段 报错问题：
```text
- error: error returned from database: column "regdate" is of type integer but expression is of type text
```

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/278153/160529722-c4a60d16-5cbf-4608-832d-5b90c86ff0c3.png)
![image](https://user-images.githubusercontent.com/278153/160529745-eb41d6dc-1b6b-4e59-ba88-89155807f527.png)

刚开始我以为是zhu'jian主键的问题，后面发现是Option 为none时，所有null，它当text？
